controllers/vgmanager_daemonset.go

```diff
@@ -128,13 +126,8 @@ func newVGManagerDaemonset(r *LVMClusterReconciler, ctx context.Context, lvmClus
 	// try to get vgmanager image from env and on absence get from running pod
 	// TODO: investigate why storing this env in a variable is failing tests
 	image := os.Getenv("VGMANAGER_IMAGE")
```

Comment:
I think this can be removed and image can be set to r.ImageName

Reply:
can i take this up during removal of standalone vg_manager?